### DISCUSSION TITLE
Add Facebooks' Seamless m4t translation models?

### DISCUSSION THREAD

**Discussion by FemBoxbrawl** [rsxdalv/TTS-WebUI#231] [Ideas]
I think that would be a great addition to the WebUI. This would be cool

**Comment by FemBoxbrawl** [comment]
not just models but like a whole tab i mean

**Reply by rsxdalv** [reply]
I added a demo version of it, specifically, SeamlessM4T v2 text to speech translation.
https://github.com/rsxdalv/tts-generation-webui/pull/256
I'm not sure. It's very interesting but I tried some English to Japanese and while it did contain some of the things that should have been said, it was a bit of a wordsalad a bit of a soundsalad.

If there's an use case/anything, I can work on expanding the functionality.

**Comment by rsxdalv** [comment]
Oh, it seems that it is no longer licensed with a CC BY NC, I might be able to include it without jeopardizing the whole project.

**Reply by rsxdalv** [reply]
Reopened the issue for it https://github.com/rsxdalv/tts-generation-webui/issues/146
